src/providers/ner-tagger.php

```diff
@@ -1,12 +1,71 @@
 <?php
 
 namespace SearchApi\Providers;
-
+use SearchApi\Models\Keyword;
 use SearchApi\Services\Tagger;
+use StanfordNLP\NERTagger as StanfordNERTagger;
 
 /**
- * NerTagger - This is a tagger implementation that uses the Named Entity Recognizer
+ * NerTagger - This is a tagger implementation that uses the Named Entity Recognizer and returns an array of keywords.
  * @seeAlso: http://nlp.stanford.edu/software/CRF-NER.shtml
  */
 class NerTagger implements Tagger {
+  // Convert a search string into an array of Keyword objects
+  public function get_tags( $request_string = '' ) {
+    // Call the NER Tagger service
+    $tagger_results = $this->tagger_service( $request_string );
+    // Convert results into keywords
+    $keywords = $this->results_to_keywords( $tagger_results );
+
+    return $keywords;
+  }
+
+  // Call the NER service and save results as a string containing an XML document
+  public function tagger_service( $request_string = '' ) {
+    if ( $request_string === null || $request_string === '' ) {
```

Comment:
This is the last thing I would like to see changed, otherwise its good to merge!